.github/workflows/ci.yaml

```diff
@@ -38,6 +38,7 @@ jobs:
       - name: Build
         run: |
           set -o pipefail
-          ./mvnw clean deploy ${{ env.MVN_COMMON_OPTIONS }} \
+          
+          mvn clean deploy ${{ env.MVN_COMMON_OPTIONS }} \
```

Comment:
could you please share more info abouth why do we revert the `mvnw` command

Reply:
Maven Wrapper ensures that the project's build and dependency management use a specific version of Maven. However, the Amoro shade project currently does not have this requirement, so we may not need to introduce it.